Assets/Python/GreatPeople.py

```diff
@@ -275,40 +336,70 @@ def setup():
 		"Jamshetji Tata", # 19th
 		"Ardeshir Godrej", # 19th
 		iGlobal,
+		"V. O. Chidambaram Pillai", # 20th
 		"fIndra Nooyi", # 20th
 	],
 	iGreatEngineer : [
 		"Baudhayana", # 8th BC
 		"Lagadha", # 1st
+		iMedieval, 
+		"Gundan Anivaritachari", # 7th
+		iRenaissance, 
+		"Vidyadhar Bhattacharya", # 18th
 		iIndustrial,
 		"Jagadish Chandra Bose", # 19th
+		"Mokshagundam Visvesvaraya", # 19th
 		iGlobal,
-		"Chandrasekhara Venkata Raman", # 20th
+		"fEulie Chowdhury", # 20th
+		"Satish Dhawan", # 20th
+		"Charles Correa", # 20th
 	],
 	iGreatStatesman : [
 		"Chanakya", # 4th BC
 		iMedieval,
+		"Amoghavarsha", # 9th
+		"Chavundaraya", # 10th
 		"Rajaraja Chola", # 10th
 		iRenaissance,
-		"Krishna Devaraya", # 16th
-		"Shivaji Bhosle", # 17th
-		"Tipu Sultan", # 18th
+		"Ariyanatha Mudaliar", # 16th
+		"Nana Fadnavis", # 18th
+		"fBegum Samru", # 18th
 		iIndustrial,
 		"Ram Mohan Roy", # 19th
 		"Ranjit Singh", # 19th
 		iGlobal,
+		"fSarojini Naidu", # 20th
+		"Sarvepalli Radhakrishnan", # 20th
 		"Jawaharlal Nehru", # 20th
 	],
 	iGreatGeneral : [
 		"Chandragupta Maurya", # 4th BC
 		"Samudragupta", # 4th BC
 		iMedieval,
-		"Rajaraja Chola", # 12th
+		"Dhruva Dharavarsha", # 8th
+		"Mihira Bhoja", # 9th
+		"Rajaraja Chola", # 10th
 		iRenaissance,
 		"fRani Durgavati", # 16th
-		"Shivaji Bhosle", # 17th
```

Comment:
Why did you remove him? The LH is Shahuji.

Reply:
Interesting. The XML tags say Shivaji, which is what I had checked, plus the fact that Shivaji seemed like a more important figure. The civilopedia also has a piece on Shivaji, not on Shahuji. The XML text however is Shahuji, and so is the constant in Consts.py. I'll put Shivaji back in the GP list, but what is the intention here?